addons/account_bank_statement_import/wizard/journal_creation.py

```diff
@@ -6,37 +6,29 @@ class account_bank_statement_import_journal_creation(models.TransientModel):
     _name = 'account.bank.statement.import.journal.creation'
     _description = 'Import Bank Statement Journal Creation Wizard'
 
-    name = fields.Char('Journal Name', required=True)
-    currency_id = fields.Many2one('res.currency', 'Currency', readonly=True)
-    account_number = fields.Char('Account Number', readonly=True)
+    name = fields.Char(string='Journal Name', required=True)
+    currency_id = fields.Many2one('res.currency', help='The currency used to enter statement', string="Currency", readonly=True)
+    company_id = fields.Many2one('res.company', string='Company', required=True, default=lambda self: self.env.user.company_id,
+        help="Company this account belongs to.")
+    bank_id = fields.Many2one('res.bank', string='Bank')
```

Comment:
We can use only bank_account_id if it's ok to open a second wizard to create the bank account (the account number is set, but user should be able to choose account type / bank / stuff added by other module)